platform/core/PolyPodCoreAndroid/core/build.gradle

```diff
@@ -0,0 +1,46 @@
+plugins {
+    id 'com.android.library'
+    id 'kotlin-android'
+    id 'org.jlleitschuh.gradle.ktlint' version '10.2.0'
+}
+
+ktlint {
+    debug = true
+    verbose = true
+    reporters {
+        reporter "checkstyle"
+        reporter "plain"
+    }
+    filter {
+        exclude("**/FlatbuffersGenerated/**")
```

Comment:
Let me see if I understand what you're doing here. Essentially, you're excluding a specific path from ktlint, which is done at the top level. So is this really used? Wouldn't you need to do this at the top level configuration?

Reply:
I don't quite understand what you mean by `top level configuration`?

Reply:
The `ktlint` configuration we (might) have at the root or `platform/android` directory. We don't actually have such thing, mainly because we don't need it... But if we did, my point is that possibly that should be included in the root directory somehow so that it's actually picked up by ktlint and its workflow.

Reply:
I understand, make sense.
As for `is it used` - I missed to run `ktlint` workflow also for android core. I'll add it.

Reply:
Just tested https://github.com/polypoly-eu/polyPod/runs/6466721501?check_suite_focus=true and ktlint was executed on the android core.